mutator-common/src/report.rs

```diff
@@ -154,7 +154,7 @@ impl Report {
     /// Prints the report to stdout in a table format.
     pub fn print_table(&self) {
         let mut builder = Builder::new();
-        builder.push_record(["Module", "Mutants tested", "Mutants killed", "Percentage"]);
+        builder.push_record(["Module", "Mutants tested", "Mutants killed", "Coverage"]);
```

Comment:
I'd keep it as it was 😄

Reply:
I will revert it in the next PR so we don't waste CI resources 😄